lib/util/src/vlb.c

```diff
@@ -125,21 +128,26 @@ vlb_free(void *mem, size_t used)
 merr_t
 vlb_init(void)
 {
-    int i;
+    if (vlb_initialized)
+        return 0;
```

Comment:
When do we call `vlb_init` more than once?

Reply:
Early on, from hse_platform_init()...

Reply:
@alexttx no where but this is just the pattern I see us use.

Reply:
I misread your question.  This is so that he can call vlb_fini() if hse_platform_fini() fails in the middle and vlb_fini() will do the right regardless of where it fails, if I'm reading this correctly...